cornflow-server/Dockerfile

```diff
@@ -8,6 +8,9 @@ LABEL maintainer="sistemas@baobabsoluciones"
 ENV DEBIAN_FRONTEND noninteractive
 ENV TERM linux
 
+# CORNFLOW vars
+ARG CORNFLOW_VERSION=1.0.0a6
+
 # install linux pkg
 RUN apt update -y && apt-get install -y --no-install-recommends \
```

Comment:
here it's necessary to install ssh

`RUN apt update -y && apt-get install -y --no-install-recommends gcc ssh`